src/test/java/reposense/git/GitLsTreeTest.java

```diff
@@ -71,11 +71,12 @@ public void windows_cloneInvalidWindowsFilePaths_throwsInvalidFilePathException(
     }
 
     @Test
-    public void unix_cloneInvalidWindowsFilePaths_success() {
+    public void unix_cloneInvalidWindowsFilePaths_success() throws Exception {
         // Runs test only on non Windows (Unix) operating systems
         Assume.assumeTrue(!SystemUtil.isWindows());
 
         config.setBranch("391-invalid-filepaths");
+        validateFilePaths(config);
```

Comment:
hmmm.. but the validateFilePaths method checks for filepaths compatible to windows tho. This TC is for unix path.
<img width="796" alt="Screen Shot 2020-04-04 at 1 49 08 AM" src="https://user-images.githubusercontent.com/32864116/78390067-7acf8580-7616-11ea-8508-44dbb3e197da.png">

Reply:
Sorry, could you clarify what you mean by the test case being for unix paths?

From what I understand - this test case is checking whether a repo that contains file paths that are invalid on Windows, can pass the validation step in `GitLsTree.validateFilePaths`

Reply:
Ah I see. 

I was a lil confused with the test case above, it says that it runs on non windows (unix) OS-es, while the file paths are invalid **Windows** file paths. So I thought it should have been invalid Unix file paths instead.

Reply:
Yes, perhaps the branch that contains the test files should be named `invalid-windows-filepaths` instead